app/controllers/ideas_controller.rb

```diff
@@ -1,5 +1,6 @@
 class IdeasController < ApplicationController
   def index
+  	@ideas = Idea.all
```

Comment:
@korneliacieslik 
Here you have too much spaces, it's minor thing, but for better readability and being consistent, would be great if you fix it :)

What do you think about it? :)
```
  def index
    @ideas = Idea.all
  end
```